WEB/src/components/Activities/ChooseActivities.vue

```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+  import { ref, onMounted } from 'vue'
+
+  const selectedActivity = ref(0)
+  const isThereAnyActivitySelected = ref(false)
+
+  const listActivities = [
+    { 
+      activity_name: 'FlashCards',
+      url: '/src/assets/logo/flashcards.svg',
```

Comment:
When giving the path of an image, please reference the "@" alias for the "/src" folder instead of the relative or absolute path because when we deploy the app, we may encounter issues with quick! [REF]https://stackoverflow.com/questions/66419471/vue-3-vite-dynamic-img-src)